detectron2/modeling/backbone/resnet.py

```diff
@@ -361,7 +362,7 @@ def __init__(self, stem, stages, num_classes=None, out_features=None):
 
         if num_classes is not None:
             self.avgpool = nn.AdaptiveAvgPool2d((1, 1))
-            self.linear = nn.Linear(curr_channels, num_classes)
+            self.linear = Linear(curr_channels, num_classes)
```

Comment:
could you undo this since we know it will not happen to backbone.
We're eventually going to change back in the future anyway when moving to higher version of pytorch.

Reply:
I thought linear should be also replaced because conv2d is replaced, i will undo this!